### PR TITLE
feat: Use standard terms for Airflow Connection to Firebolt

### PR DESCRIPTION
**Pull request by yuryfirebolt**

### CODE REVIEW

File: firebolt_provider/hooks/firebolt.py [hunk 1]
```diff
@@ -50,15 +50,20 @@ def get_ui_field_behaviour() -> Dict:
...
-                "login": "firebolt userid",
-                "password": "password",
+                "host": "The host name of your Firebolt database",
+                "schema": "The database to connect to",
+                "login": "The email address you use to log into Firebolt",
+                "password": "Your password to log in to Firebolt",
                 "extra": json.dumps(
                     {
-                        "engine_name": "firebolt engine name",
+                        "account_name": "firebolt",
```
Comment (ptiurin) [inline firebolt_provider/hooks/firebolt.py:65]: Should this contain a hint on what account name is, like other fields?

File: tests/hooks/test_firebolt.py [hunk 2]
```diff
@@ -51,6 +51,7 @@ def test_get_conn(self, mock_connect):
             api_endpoint="api_endpoint",
             database="firebolt",
             engine_name="test",
+            account_name="firebolt"
```
Comment (ptiurin) [inline tests/hooks/test_firebolt.py:54]: lets add a test for `engine_name=None` as well.

**Review by ptiurin — COMMENTED** [review]
Apart from those comments, looks good.

File: firebolt_provider/hooks/firebolt.py (hunk 1 above)
Comment (yuryfirebolt) [reply firebolt_provider/hooks/firebolt.py]: yep, changed it

File: tests/hooks/test_firebolt.py (hunk 2 above)
Comment (yuryfirebolt) [reply tests/hooks/test_firebolt.py]: will add

**Review by ptiurin — APPROVED** [review]
LGTM
